cleverhans/model_zoo/deep_k_nearest_neighbors/dknn.py

```diff
@@ -413,8 +550,18 @@ def get_activations(data):
 def main(argv=None):
   assert dknn_tutorial()
 
-
-if __name__ == '__main__':
+def define_default_argument_values():
```

Comment:
Could you remove `define_default_argument_values` and leave the flags declarations as they were previously for consistency with the rest of the library?